src/Client/ConnectionPoolWithFailover.cpp

```diff
@@ -79,14 +79,6 @@ IConnectionPool::Entry ConnectionPoolWithFailover::get(const ConnectionTimeouts
     return Base::get(max_ignored_errors, fallback_to_stale_replicas, try_get_entry, get_priority);
 }
 
-Priority ConnectionPoolWithFailover::getPriority() const
-{
-    return (*std::max_element(nested_pools.begin(), nested_pools.end(), [](const auto & a, const auto & b)
-    {
-        return a->getPriority() < b->getPriority();
-    }))->getPriority();
-}
-
```

Comment:
we don't need this function?

Reply:
This `getPriority()` is about getting a cluster node priority from config. I can't picture how this method can be helpful for `ConnectionPoolWithFailover`. If anybody needs to get a cluster node with the highest priority in config then this functionality should be just a separate function and there is no need for pure virtual method in `IConnectionPool` for this